tests/migrations/test_autodetector.py

```diff
@@ -657,6 +674,14 @@ def test_alter_field(self):
         self.assertOperationTypes(changes, 'testapp', 0, ["AlterField"])
         self.assertOperationAttributes(changes, "testapp", 0, 0, name="name", preserve_default=True)
 
+    def test_partial_doesnt_detect_changes_when_unchanged(self):
+        """#26475 - Tests functools.partial serialisation always showing changes."""
```

Comment:
I'll inline them, as any other tests are likely to want slightly different states.